src/components/Pages/Unit/widgets/UnitMaterials/DerivedUnits.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table, List } from 'semantic-ui-react';
+import { withNamespaces } from 'react-i18next';
+
+import { NO_NAME, CT_VIDEO_PROGRAM_CHAPTER, CT_CLIP } from '../../../../../helpers/consts';
+import { CollectionsBreakdown } from '../../../../../helpers/mdb';
+import { canonicalLink } from '../../../../../helpers/links';
+import { ellipsize } from '../../../../../helpers/strings';
+import Link from '../../../../Language/MultiLanguageLink';
+import UnitLogo from '../../../../shared/Logo/UnitLogo';
+import { FrownSplash } from '../../../../shared/Splash/Splash';
+
+export const derivedUnitsContentTypes = [CT_VIDEO_PROGRAM_CHAPTER, CT_CLIP];
+
+const renderUnit = (unit, t) => {
+  const breakdown = new CollectionsBreakdown(Object.values(unit.collections || {}));
```

Comment:
1. This is exact copy from Sectoins/Programs/tabs/Clips/List.js Please remove it to a helper
2. There are a lot of exported functions renderUnit. Please remove "export" and check whether somebody imports them. All really imported ones move to the aforementioned helper